sheetstorm/management/commands/generate_test_data.py

```diff
@@ -61,6 +75,42 @@ def _set_user_factory_parameters(user_type):
             "is_superuser": user_type == superuser_user_type,
         }
 
+    @staticmethod
+    def _get_project_options(options):
```

Comment:
Too general method name. You're saying here that you're giving some kind of options in this method but from method body I can see that you're giving some numbers for projects - probably number how many projects should be created. Please be more precise in method name - if method does more things and it is hard to name this method please extract it to additional methods

Reply:
Done